config/api.yml

```diff
@@ -0,0 +1,9 @@
+:collections:
+  :migration_analytics_manifest:
+    :description: Migration Analytics Manifest
+    :options:
+    - :arbitrary_resource_path
+    - :collection
+    :identifier: migration_analytics_manifest
+    :verbs:
+    - :get
```

Comment:
Missing the:

```
:collection_actions:
  :get:
  - :name: read
    :identifier:  <miq_product_feature>
```

section. Really need an miq_product_feature, so that only users with a certain role are allowed to run
this api, otherwise it's open to all users.

I didn't see a db/fixtures/miq_product_features.yml in this PR.  Are we leveraging product features established by the core Miq or cfme-cloud_services maybe (i.e. red_hat_cloud_services) ?  
```

Reply:
Thanks @abellotti, I wasn't sure what that collection_actions section was for, since this isn't a real database collection. I'm adding it now with the feature red_hat_migration_analytics, which you're right, we do need to define somewhere...

@agrare do you think this should be its own miq_product_feature, or should it just share permissions with the red_hat_cloud_services feature? It's just a disconnected use case for functionality from that plugin, and produces data to be used on cloud.redhat.com, so maybe it should be considered part of that feature even though it's in its own plugin.

Reply:
For now I'm actually going to just push a commit that uses the red_hat_cloud_services feature, if we need it to define its own feature I can add that yml file in another commit.

Reply:
@mturley probably best if we have our own product feature

Reply:
Sounds good @agrare. I just added the `red_hat_migration_analytics` feature and set it as the identifier on this collection_action.

Reply:
(oops, nevermind.. this was merged before I pushed that commit. I'll make another PR for it.)

Reply:
https://github.com/RedHatCloudForms/cfme-migration_analytics/pull/10